### DISCUSSION TITLE
Can hub's name be customized?

### DISCUSSION THREAD

**Discussion by TechnicBRICKs** [pybricks/support#248] [Q&A] [answered]
Is there a way to customize the hub name, as it appears in the window with the list of available hubs to pair?

![image](https://user-images.githubusercontent.com/67207661/106529462-6a13c880-64e2-11eb-8d0b-ea151b124c62.png)

**Comment by dlech** [comment, marked as answer]
Not yet... (on the todo list)

**Reply by TechnicBRICKs** [reply]
Thanks!

**Reply by dlech** [reply]
This is now possible.

**Comment by dlech** [comment]
This feature is being tracked in #52.
